engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/Utils.scala

```diff
@@ -22,24 +18,25 @@ object Utils extends LazyLogging {
     case t: Throwable => logger.error("Error occurred, but skipping it", t)
   }
 
-  def gracefulStopActor(actorRef: ActorRef, actorSystem: ActorSystem): Unit = {
+  def stopActorAndWaitUntilItsNameIsFree(actorRef: ActorRef, actorSystem: ActorSystem): Unit = {
     import actorSystem.dispatcher
-    logger.info(s"Gracefully stopping $actorRef")
+    logger.info(s"Stopping $actorRef")
 
-    val gracefulStopFuture = for {
-      _ <- gracefulStop(actorRef, GracefulStopTimeout)
+    actorSystem.stop(actorRef)
```

Comment:
Why so brutal stop? Shouldn't we try with graceful stop and after some timeout eventually perform less graceful stop? We need at least some comment with explanation to stop someone from refactoring it to the previous version ;-)

Reply:
this stop still allows it to finish processing current message, and it's sufficient as I explained here: https://github.com/TouK/nussknacker/pull/6314#issuecomment-2208580681

I could add some comments to the code i guess